sdcm/nemesis.py

```diff
@@ -670,11 +669,9 @@ def disrupt_nodetool_enospc(self, sleep_time=30, all_nodes=False):
             nodes = [self.target_node]
         self._set_current_disruption('Enospc test on {}'.format([n.name for n in nodes]))
 
+        group_common_db_events = GroupCommonDbEvents()
```

Comment:
why in some places you are putting it into a variable, and in some place you are not ?
also not total clear to me why it's need to be a class, and can't be function on a module ?

Reply:
To "save" the second object...